### PR TITLE
Update eslint

### PR DESCRIPTION
**Pull request by andykwan1124**
# Description

Fix lint errors and added `yarn lint` command

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Works on staging site

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kozr** [comment]
thx for fix lint

File: src/components/faqTemplates/shared/FaqList.jsx [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-      </div>
-    )) ?? null
-  )
-}
+const ExpandableFaqList = ({ list }) =>
+  list?.map(({ question, answer }) => (
+    <div key={question}>
+      <ExpandableFaq question={question} answer={answer} />
+    </div>
+  )) ?? null
```
Comment (kozr) [inline src/components/faqTemplates/shared/FaqList.jsx:9]: wow this code is so good

Comment (andykwan1124) [reply src/components/faqTemplates/shared/FaqList.jsx:9]: I wonder who wrote it

File: src/lib/Helpers.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prefer-rest-params */
```
Comment (kozr) [inline src/lib/Helpers.jsx:1]: @rmehri01 when you have time, can you get rid of this line and work on the errors?

**Review by kozr — APPROVED** [review]

File: src/lib/Helpers.jsx (hunk 2 above)
Comment (rmehri01) [reply src/lib/Helpers.jsx:1]: Sounds good!

**Pull request merged**